pkgs/os-specific/darwin/apple-source-releases/Csu/default.nix

```diff
@@ -1,7 +1,10 @@
-{ lib, appleDerivation', stdenv }:
-
-appleDerivation' stdenv {
+{ lib
+, Libsystem
```

Comment:
This isn't used and I don't think it serves the same unconditional override purpose?